lightning/src/ln/channelmanager.rs

```diff
@@ -4019,6 +4019,10 @@ impl<Signer: Sign, M: Deref, T: Deref, K: Deref, F: Deref, L: Deref> ChannelMana
 					pending_events.push(events::Event::PendingHTLCsForwardable {
 						time_forwardable: time
 					});
+				} else {
```

Comment:
Heh, sorry, the naming here is a bit confusing. We may want to rename `PendingHTLCsForwardable`, but what it really means is "we have some HTLC *events* that need forwarding, but for some vague sense of privacy we want to batch them and do them after a little bit of time, so here's an event to do it later". Thus, I don't think this should be in an `else` branch.